pom.xml

```diff
@@ -250,10 +210,15 @@
         <dependency>
             <groupId>org.easymock</groupId>
             <artifactId>easymock</artifactId>
-            <version>3.6</version>
+            <version>4.3</version>
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>1.7.7</version>
+        </dependency>
```

Comment:
Why have we gained another logging framework as a dependency? Is it that we can no longer transitively depend on it via the log4j interface to slf4j?

Reply:
We used to get this transitively from `jgit`. The new version of `jetty-servlets` brings in version 2.x which supersedes that and spews forth error messages when used with `log4j` at the moment due to [this](https://issues.apache.org/jira/browse/LOG4J2-3139).

Another option would be excluding `slf4j-api` from `jetty-servlets`, so the `jgit`-imported version is effective once again. Both approaches seem to be practically equivalent, but I think the explicit import is preferable.